test/unit/compute_resources/ovirt_test.rb

```diff
@@ -24,4 +24,29 @@ class OvirtTest < ActiveSupport::TestCase
       assert_find_by_uuid_raises(ActiveRecord::RecordNotFound, cr)
     end
   end
+
+  describe "associating operating system" do
+    setup do
+      @operating_systems_xml = Nokogiri::XML(File.read('test/fixtures/ovirt_operating_systems.xml'))
```

Comment:
We just care that `available_operating_systems` returns an OS definition to make the match. How about just stubbing `@compute_resource.stubs(:available_operating_systems)` instead of reading an XML file on every test? That should be much faster (instantiating a string vs parsing an XML file from disk) and we won't have to maintain a fixture that might change depending on the oVirt version.